datasource/ethereum/src/ethereum_adapter.rs

```diff
@@ -366,11 +366,6 @@ where
                         1 => {
                             let mut block_ptrs = vec![];
                             for log in chunks[0].iter() {
-                                if block_ptrs.len() >= 100 {
-                                    // That's enough to process in one iteration
-                                    break;
-                                }
-
```

Comment:
This chunking now happens in `load_blocks` in `block_stream.rs`

Reply:
Noted!